geojson_pydantic/features.py

```diff
@@ -1,13 +1,21 @@
-from typing import Union, Dict, List,  Optional, Any
+from typing import Union, Dict, List, Optional, Any
 from pydantic import BaseModel, Field, validator
 
 from .utils import BBox
-from .geometries import Point, MultiPoint, LineString, MultiLineString, \
-    Polygon, MultiPolygon
+from .geometries import (
+    Point,
+    MultiPoint,
+    LineString,
+    MultiLineString,
+    Polygon,
+    MultiPolygon,
```

Comment:
I think these imports are unused now

Reply:
Looks like it. I can remove them. Adding mypy and/or pylint in github actions could help detect this in the future.

Reply:
yup, just put up https://github.com/developmentseed/geojson-pydantic/pull/10